examples/geostrophic_adjustment.jl

```diff
@@ -38,26 +45,26 @@ coriolis = FPlane(f=1e-4)
 
 model = HydrostaticFreeSurfaceModel(grid = grid,
                                     coriolis = coriolis,
-                                    free_surface=ImplicitFreeSurface())
+                                    free_surface = ImplicitFreeSurface())
 
 # ## A geostrophic adjustment initial value problem
 #
 # We pose a geostrophic adjustment problem that consists of a partially-geostrophic
-# Gaussian height field complemented by a geostrophic ``y``-velocity,
+# gaussian height field complemented by a geostrophic ``y``-velocity,
```

Comment:
Gaussian is better I think as it's a proper name, but there are many other conventions so please don't feel you need to change it.  Just my two cents.

Reply:
Yeah, it was those pesky other conventions that I was bending too. But maybe we should use "Gaussian" in text even if we use `gaussian` in code.

Reply:
I think that would be better but this is a minor detail and not worth losing sleep over.